.github/workflows/xrt_master.yml

```diff
@@ -1,14 +1,15 @@
-name: XRT Master
+name: XRT 2024.1
```

Comment:
Do you mean that `master` is no longer tested?
Should the file be renamed?

Reply:
Hi @keryell This would be similar to master for 2024.1 branch and current master is 2024.2 so there will be another file for that